places/tony-chaska.md

```diff
@@ -0,0 +1,13 @@
+## Chaska, MN
+
+### Fun Facts
+- It is cold in the winter.
+
+### Places to Eat
+- Home
+
+### Places to Go
+- Florida
+
+### Places to Avoid
+- Outdoors in the Winter.
```

Comment:
I love the cold!!  But then again I live in California.  Not sure if I know what cold is.  ❄️